qiita_pet/templates/study_ajax/artifact_file_selector.html

```diff
@@ -245,7 +245,7 @@
   </div>
   {% for ft, req, files in file_types %}
     <div class="col-md-3" id="{{ft}}-div">
-      <input type="hidden" name="{{ft}}" id="{{ft}}" value="" class="fileListParam">
+      <input type="hidden" name="{{ft}}[]" id="{{ft}}" value="" class="fileListParam">
```

Comment:
What is the [] used for? Is it to make the name of the input element different?

Reply:
`[]` will make it an array vs as a string, and missing it is why files with spaces will fail cause JS tries to parse it incorrectly

Reply:
Thanks!